OpenRA.Mods.Common/Traits/Sound/AnnounceOnSeen.cs

```diff
@@ -18,18 +20,35 @@ public class AnnounceOnSeenInfo : ITraitInfo
 		[Desc("Should there be a radar ping on enemies' radar at the actor's location when they see him")]
 		public readonly bool PingRadar = false;
 
-		public readonly string Notification = "EnemyUnitSighted";
+		public readonly string Notification = null;
```

Comment:
Did anything rely on this default value? I can't find a reference to it anywhere.